utils/luci-app-owm/files/owm.lua

```diff
@@ -53,6 +53,15 @@ function unlock()
 	os.execute("rm -f "..lockfile)
 end
 
+-- a location for the node is required to run
+if next(owm.get_position())==nil then
```

Comment:
an empty table is not NIL, so this "next($tablevar)" can detect this. get_position()==nil will always be true.

this kind of test only needs 6bytes on disc, checking the values of long & lat and return nil or the data as table will cost more flash-space